examples/custom-prisma-driver-adapter/README.md

```diff
@@ -0,0 +1,28 @@
+## Example Project - Extend Prisma Schema
+
+This project implements an example of mutating the Prisma Schema to:
+
+- Set the Prisma Binary Target;
+- Add a multi-column column unique constraint to a list; and
+- Add a NOT NULL relationship field
+
+These show three separate ways of mutating the Prisma Schema see `keystone.ts` and `schema.ts`.
+
+## Instructions
+
+To run this project, clone the Keystone repository locally, run `pnpm` at the root of this repository, then navigate to this directory and run:
+
+```shell
+pnpm dev
+```
+
+This will start the Admin UI at [localhost:3000](http://localhost:3000).
+You can use the Admin UI to create items in your database.
+
+You can also access a GraphQL Playground at [localhost:3000/api/graphql](http://localhost:3000/api/graphql), which allows you to directly run GraphQL queries and mutations.
+
+Congratulations, you're now up and running with Keystone! 🚀
+
+## Try it out in CodeSandbox 🧪
+
+You can play with this example online in a web browser using the free [codesandbox.io](https://codesandbox.io/) service. To launch this example, open the URL <https://githubbox.com/keystonejs/keystone/tree/main/examples/extend-prisma-schema>. You can also fork this sandbox to make your own changes.
```

Comment:
This whole directory (custom-prisma-driver-adapter) is the copy of the `extend-prisma-schema` example.

This could also be the start of showing an example with prisma client extensions like https://github.com/prisma/extension-read-replicas

would be super beneficial for users

Reply:
I am thinking we might use Prisma extensions for hooks in the near future, so it does have me thinking about how this functionality might impact the order of operations there.

As is, if we supported the custom `PrismaClient`, Keystone would be `.extend`ing the custom client, which I think is ok?

Reply:
> I am thinking we might use Prisma extensions for hooks in the near future, so it does have me thinking about how this functionality might impact the order of operations there.

I love this idea would make the usage of `context.prisma` over `context.db` much easier if you have custom logic in list hooks